CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## x.x.x yyyy-mm-dd
+### Payments
+* [Fixed] Including inferred mandate data for Paypal.
```

Comment:
I don't think it's a fix so much as a new feature - enabling SI and PI+SFU support for Paypal in the API bindings.

Reply:
Or actually this PR is just adding SI support, maybe - but we should test the flow to make sure.

Reply:
@charliecruzan-stripe you are currently trying to do this, correct? And making this change made it work for you?